core/actions/buildJournalSite/action.ts

```diff
@@ -0,0 +1,28 @@
+import { Book } from "lucide-react";
+import * as z from "zod";
+
+import { Action } from "db/public";
+import { FileText } from "ui/icon";
+
+import { fieldName } from "../_lib/zodTypes";
+import { defineAction } from "../types";
+
+const schema = z.object({
+	token: z.string().describe("The token for the site builder"),
+	articles: fieldName(),
+	collections: fieldName(),
+	mainColor: fieldName(),
+	accentColor: fieldName(),
+});
```

Comment:
these are not actually used, but we should do so in the future (or find some other way to configure it)